README.md

```diff
@@ -60,38 +175,25 @@ When developing one of the micro front-end applications you will therefore have
 Given this complexity, it is recommended that you use a tool like [iTerm2](https://iterm2.com) (on Mac) or an equivalent terminal shell on Windows to make terminal management simpler. iTerm2 allows you to setup a pre-defined window layout of terminal sessions, including the directory in which the session starts. This setup, along with simple shell scripts in each project that configure and start the environment, will allow you to get your development environment up and running quickly and easily.
 
 ## Git
+
 ### Branching
 When working on Jira tickets, we link associated Git PRs and branches to the tickets. Use the following naming convention for branches:
 
 `[TICKET #]_short-description`
 
 e.g.: `PROD-1516_work-issue`
 
+The overall base branch is `develop`. Whenever a new epic is created and started working on, then a new epic branch is checked out from `develop` branch(for eg. like this `PROD-120_find-me-data`, in this case `PROD-120` is an epic). When a task/bug is assigned to a developer within a particular epic and when the developer starts working on the task/bug then the developer has to consider the epic branch(`PROD-120_find-me-data`) as base branch and create a new feature branch based on the above mentioned branch naming convention.
```

Comment:
@brooketopcoder @testflyjets Please let me know if it make sense to mention this in readme.

Reply:
That's probably reasonable to add to the ticket, assuming we don't change that strategy anytime soon.

Reply:
ok cool 👍

Reply:
@hentrymartin the epic branch for PROD-120 was a one-off bc the develop branch had a bunch of changes merged that we didn't want to QA. so we created a separate branch that had only the subset of changes for find me data.

i think for now, let's just add a TBD for this section and plan to get the entire team together to iron this out.